Example/WalletApp/DomainLayer/Chat/ChatService.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import Combine
+import WalletConnectChat
+import WalletConnectRelay
+
+typealias Stream<T> = AsyncPublisher<AnyPublisher<T, Never>>
+
+final class ChatService {
```

Comment:
same,
I think if we ever integrate chat into walletApp it will be done with web3inbox